.github/workflows/publish_flutter.yaml

```diff
@@ -0,0 +1,20 @@
+# A CI configuration to auto-publish pub packages.
+
+name: Publish Flutter Packages
+
+on:
+  pull_request:
+    branches: [ master ]
+  push:
+    tags: [ '[A-z]+-v[0-9]+.[0-9]+.[0-9]+' ]
+
+jobs:
+  publish:
+    if: ${{ github.repository_owner == 'dart-lang' }}
+    uses: dart-lang/ecosystem/.github/workflows/publish.yaml@main
+    with:
+      use-flutter: true
```

Comment:
Seems like something we should auto-detect, right? Maybe?

Reply:
It did initially, and then that functionality was removed.
https://github.com/dart-lang/ecosystem/pull/158

I'm not sure I understand the "We need the flag anyhow to decide whether to setup Flutter in the first place" statement. @mosuem do you recall what this meant?

Reply:
It means that instead of setting up dart via `setup-dart`, we would use the `setup-flutter` action.